src/main/java/net/sf/jabref/gui/FindUnlinkedFilesDialog.java

```diff
@@ -337,15 +354,13 @@ private void storeLastSelectedDirectory(File lastSelectedDir) {
      *
      * @return The persistently stored working directory path for this view.
      */
-    private File loadLastSelectedDirectory() {
+    private Path loadLastSelectedDirectory() {
         String workingDirectory = Globals.prefs.get(FindUnlinkedFilesDialog.GLOBAL_PREFS_WORKING_DIRECTORY_KEY);
         if (workingDirectory == null) {
             workingDirectory = Globals.prefs.get(JabRefPreferences.WORKING_DIRECTORY);
         }
-        lastSelectedDirectory = new File(workingDirectory);
-        if (lastSelectedDirectory.isFile()) {
-            lastSelectedDirectory = lastSelectedDirectory.getParentFile();
-        }
+        lastSelectedDirectory = Paths.get(workingDirectory);
```

Comment:
The condition has been removed which may cause trouble. I think using the folder if a file has been deleted should still be done.